tests/tracer/test_ext.py

```diff
@@ -21,7 +25,154 @@ def _updateenv(monkeypatch, env):
         monkeypatch.setenv(str(k), str(v))
 
 
+@pytest.fixture
+def git_repo(tmpdir):
+    cwd = str(tmpdir)
+    subprocess.check_output("git init", cwd=cwd, shell=True)
+    subprocess.check_output('git remote add origin "git@github.com:test-repo-url.git"', cwd=cwd, shell=True)
+    subprocess.check_output('git config --local user.name "John Doe"', cwd=cwd, shell=True)
+    subprocess.check_output('git config --local user.email "john@doe.com"', cwd=cwd, shell=True)
+    subprocess.check_output("touch tmp.py", cwd=cwd, shell=True)
+    subprocess.check_output("git add tmp.py", cwd=cwd, shell=True)
+    subprocess.check_output(
+        'git commit --date="2021-01-19T09:24:53-0400" -m "this is a commit msg" --no-edit', cwd=cwd, shell=True
+    )
+    subprocess.check_output(
```

Comment:
do we need any tests that have different commit messages, dates or messed up email/names?

Reply:
Do you have anything in particular in mind? I think it's safe to assume we can trust git for returning accurate/correct commit messages/logs since all we are doing is passing the git values along.

Reply:
not sure I do, I think I asked in a previous comment as well, and we came to the conclusion that we'll always have something since git requires them.